### DISCUSSION TITLE
Stopped Working For Me Recently

### DISCUSSION THREAD

**Discussion by tea-money** [smasherprog/EqTool#95] [General]
The tools have worked great for me for a long while and have been fantastic. But last week the tools stopped opening suddenly. I can get the system icon in my tray, but right clicking doesn't do anything, nor do any overlays appear.
Have tried reinstalling, restarting, removing system temp files, etc but nothing seems to work.
I have to open task manager and kill the EQ Tool background process to close it.

Any thoughts why this might be happening? 
Apologies if this isn't the best place for troubleshooting.

**Comment by smasherprog** [comment]
where is the executable located?

**Reply by tea-money** [reply]
Within my EQ directory. Location of .exe hasn't changed from it working to it not working.

**Comment by smasherprog** [comment]
That does seem to be an issue. Ill look into this when i get home. Try moving the exe to a different directory

**Comment by tea-money** [comment]
Huh. That seemed to work. Deleted the old eq tool files and grabbed a fresh download - put the .exe in a different directory and it ran and updated just fine. Seems to be working as intended now, but a solution I wouldn't have thought to try (obviously)

**Comment by smasherprog** [comment]
Ya thanks for reporting this. I'll fix this issue when I get home. At work right now

**Reply by tea-money** [reply]
Happy to help. Tool is stellar. Appreciate all the hard work that goes into it.

**Comment by smasherprog** [comment]
@smasherprog Make sure to test running from in EQ directory and Ram drive

**Comment by smasherprog** [comment]
Added a message to let users know this is why pigparse wont work if it runs from the eq dir. 
Im not sure why it doesnt work, so i do a popup on startup if the program is ran from the eq dir now letting users know.
Thanks for the info
